csrankings-s.csv

```diff
@@ -1279,7 +1279,7 @@ Silvia Takahashi Rodríguez,Universidad de los Andes,https://profesores.virtual.
 Silvina L. Ferradal,Indiana University,https://luddy.indiana.edu/contact/profile/?Silvina_Ferradal,2JcHy3wAAAAJ
 Silvio Amir,Northeastern University,https://www.khoury.northeastern.edu/people/silvio-amir/,jhUQvC0AAAAJ
 Silvio Micali,Massachusetts Institute of Technology,https://people.csail.mit.edu/silvio,NOSCHOLARPAGE
-Silvio Savarese,Stanford University,http://cvgl.stanford.edu/silvio,ImpbxLsAAAAJ
+Silvio Savarese,Stanford University,https://aicenter.stanford.edu/people/silvio-savarese,ImpbxLsAAAAJ

```

Comment:
AFAICT not a full-time TT faculty member, please remove.

Reply:
Thank you for your catch